cmd/dlv/main.go

```diff
@@ -15,11 +15,16 @@ func main() {
 	if Build != "" {
 		version.DelveVersion.Build = Build
 	}
+
 	const cgoCflagsEnv = "CGO_CFLAGS"
 	if os.Getenv(cgoCflagsEnv) == "" {
-		os.Setenv(cgoCflagsEnv, "-O0 -g")
+		err := os.Setenv(cgoCflagsEnv, "-O0 -g")
+		if err != nil {
```

Comment:
os.Setenv can't fail except for a malformed key.

Reply:
True, reverted.